mne/viz/topomap.py

```diff
@@ -18,19 +18,21 @@
 
 from ..baseline import rescale
 from ..io.constants import FIFF
-from ..io.pick import pick_types, _picks_by_type
+from ..io.pick import (pick_types, _picks_by_type, channel_type, pick_info,
+                       _pick_data_channels)
 from ..utils import _clean_names, _time_mask, verbose, logger, warn
 from .utils import (tight_layout, _setup_vmin_vmax, _prepare_trellis,
                     _check_delayed_ssp, _draw_proj_checkbox, figure_nobar,
                     plt_show, _process_times)
 from ..time_frequency import psd_multitaper
 from ..defaults import _handle_default
 from ..channels.layout import _find_topomap_coords
+from ..io.meas_info import Info
 
 
 def _prepare_topo_plot(inst, ch_type, layout):
     """"Aux Function"""
-    info = copy.deepcopy(inst.info)
+    info = inst.copy() if isinstance(inst, Info) else copy.deepcopy(inst.info)
```

Comment:
Why isn't Info.copy() sufficient actually? Is there something about it that doesn't reliably get copied?